Sources/Shared/Extensions/Dictionary+Tailor.swift

```diff
@@ -91,6 +91,26 @@ public extension Dictionary {
     return property(name)
   }
 
+  /// Map value using key to String
+  ///
+  /// - Parameter name: The name of the key.
+  /// - Returns: An optional String
+  func boolean(_ name: String) -> Bool? {
+    guard let key = name as? Key else {
+      return nil
+    }
+
+    if let string = self[key] as? String {
```

Comment:
`assertFailure("Contact your backend devs because something is clearly not correct here.")`